src/common/actions/repositories.js

```diff
@@ -67,9 +68,15 @@ export function fetchUserRepositories(pageNumber) {
           if (result.pageLinks) {
             dispatch(setPageLinks(result.pageLinks));
           }
+        })
+        .catch(error => {
+          return Promise.reject(error);
         });
       })
-      .catch(error => dispatch(fetchRepositoriesError(error)));
+      .catch(error => {
+        log.warn(error);
```

Comment:
What kind of errors is it catching? Why do we want to console.log them (in production)?